lainnya/lembar-pengesahan-en.tex

```diff
@@ -0,0 +1,80 @@
+\begin{center}
+  \large
+  \textbf{APPROVAL SHEET}
+\end{center}
+
+% Menyembunyikan nomor halaman
+\thispagestyle{empty}
+
+\begin{center}
+  \textbf{\engtatitle{}}
+\end{center}
+
+\begingroup
+% Pemilihan font ukuran small
+\small
+
+\begin{center}
+  \textbf{FINAL PROJECT}
+  \\Submitted to fullfill one of the requirements for obtaining Engineering degree at Undergraduate Study Program of Computer Engineering Department of Computer Engineering Faculty of Intelligent Electrical and Informatics Technology Sepuluh Nopember Institute of Technology
```

Comment:
Should replace Computer Engineering with `\engstudyprogram{}` here. Check other sentences too.